packages/components/src/components/bal-navigation/bal-navigation.sass

```diff
@@ -1,13 +1,22 @@
 +block(nav)
-  position: fixed
+  position: absolute
   display: block
-  top: 0
-  left: 0
   width: 100%
-  transition: transform .3s ease-in-out
   z-index: 30
+  top: 0
+  left: 0
+  background-color: var(--bal-color-background)
+  //padding-bottom: 4rem
```

Comment:
remove commented padding

Reply:
re-added it as needed on mobile viewport